src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -352,6 +352,44 @@ def setUp(self):
         self.d_log = DaosLog(self.context)
         self.test_log.daos_log = self.d_log
 
+        # Add test binaries and daos binaries to PATH
+        test_dirs = {"TESTING": "tests", "install": "bin"}
+        found_path = False
+        d = os.path.dirname(os.path.abspath(__file__))
+
+        # Use the developer environment from which this python file was called
+        if ("DAOS_ENV" in os.environ) and (os.environ["DAOS_ENV"] == "dev"):
+
+            while True:
+                if os.path.basename(d) == "TESTING":
+                    added_path = os.path.join(d, test_dirs["TESTING"])
+                    os.environ["PATH"] += os.pathsep + added_path
+                    found_path = True
+                elif os.path.basename(d) == "install":
+                    added_path = os.path.join(d, test_dirs["install"])
+                    if os.path.isdir(added_path):
+                        os.environ["PATH"] += os.pathsep + added_path
+                        found_path = True
+                    else:
+                        print("ERROR: Directory does not exist: " + added_path)
+                elif re.match("^\s*\/+\s*$", d) != None:
```

Comment:
(lint) '\s'. String constant might be missing an r prefix.
(lint) '\/'. String constant might be missing an r prefix.
(lint) '\s'. String constant might be missing an r prefix.